utils/whatsapp/structs.go

```diff
@@ -0,0 +1,273 @@
+package whatsapp
+
+type WACMsgTemplating struct {
+	Template struct {
+		Name string `json:"name" validate:"required"`
+		UUID string `json:"uuid" validate:"required"`
+	} `json:"template" validate:"required,dive"`
+	Namespace string   `json:"namespace"`
+	Variables []string `json:"variables"`
+}
+
+type WACMOMedia struct {
```

Comment:
We should call these things what they are called by META in their API - not by how we use them. I also good to put some links in comments here pointing to the API docs